packages/core/components/Render/index.tsx

```diff
@@ -1,33 +1,15 @@
 "use client";
 
-import { rootDroppableId } from "../../lib/root-droppable-id";
 import { Config, Data } from "../../types/Config";
-import { DropZone, DropZoneProvider } from "../DropZone";
+import { DropZoneProvider } from "../DropZone";
+import { Render as ServerRender } from "../ServerRender";
 
 export function Render({ config, data }: { config: Config; data: Data }) {
-  // DEPRECATED
-  const rootProps = data.root.props || data.root;
-
-  const title = rootProps.title || "";
-
-  if (config.root?.render) {
-    return (
-      <DropZoneProvider value={{ data, config, mode: "render" }}>
-        <config.root.render
-          {...rootProps}
-          title={title}
-          editMode={false}
-          id={"puck-root"}
-        >
-          <DropZone zone={rootDroppableId} />
-        </config.root.render>
-      </DropZoneProvider>
-    );
-  }
-
```

Comment:
@chrisvxd  Render now uses the ServerRender component, just wrapping in context for backwards compatibility